vercel.json

```diff
@@ -49,7 +49,7 @@
     },
     {
       "source": "/flow/faq/:path*",
-      "destination": "/build/run-and-secure/faq/:path*",
+      "destination": "/references/run-and-secure/faq/:path*",
```

Comment:
Don't you need not to replace these but actually create new ones? because won't the build url ones not redirect now?

Reply:
haha just beat me to it https://github.com/onflow/docs/pull/337/commits/cff1373be2511080625dfab2c899f8de401e65ca